README.md

```diff
@@ -1,4 +1,5 @@
 # git-workflow
 
 Today, a sailor went on a walk and was happy to see the sunshine.
-The sailor felt woozy walking on land because it doesn't rock like the sea.
\ No newline at end of file
+The sailor felt woozy walking on land because it doesn't rock like the sea.
+Along came a kitty cat who jumped on the sailor's shoulders.
```

Comment:
This is the lint I created with the kitty cat.

Reply:
Oh Nice! Love Cats!